airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/BasicAcceptanceTests.java

```diff
@@ -995,7 +1000,7 @@ public void testResetAllWhenSchemaIsModified() throws Exception {
       return null;
     });
 
-    testHarness.assertSourceAndDestinationDbInSync(false);
+    testHarness.assertSourceAndDestinationDbInSync(WITHOUT_SCD_TABLE);
```

Comment:
this is nitpicky, but I don't know what `WITHOUT_SCD_TABLE` means...could we rename this to make it clearer?

Reply:
From what I understand, SCD (Slow Changing Dimensions??) tables are tables we generate for normalization. I am fairly neutral to the name, the constant I am using is the one that has been defined with the name of the parameter in the `assertSourceAndDestinationDbInSync` function.